AbletonMCP_Remote_Script/handlers/audio.py

```
@@ -0,0 +1,517 @@
+            "track_index": track_index,
+            "frozen": False,
+            "track_name": track.name,
+        }
+    except Exception as e:
+        if ctrl:
+            ctrl.log_message("Error unfreezing track: " + str(e))
+        raise
+
+
+def export_track_audio(
+    song, track_index, output_path, start_time, end_time, ctrl=None
```

Ruff (0.14.14)

[warning] Unused function argument: `start_time`

(ARG001)